testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSWalletTest.scala

```diff
@@ -375,14 +377,15 @@ object BitcoinSWalletTest extends WalletLogger {
     }
   }
 
-  private[testkit] class RandomFeeProvider extends FeeRateApi {
-    private val rnd = new scala.util.Random(TimeUtil.now.toEpochMilli)
-    private val start = 2
-    private val end = 100
+  private[bitcoins] class RandomFeeProvider extends FeeRateApi {
+    // Useful for tests
+    var lastFeeRate: Option[FeeUnit] = None
```

Comment:
Will this be problematic with Async tests? Or are we guaranteed to have one per fixture/test

Reply:
We should be guaranteed one fixture be test case